shims/spark320/src/main/scala/com/nvidia/spark/rapids/shims/spark320/SparkShimServiceProvider.scala

```diff
@@ -19,7 +19,8 @@ package com.nvidia.spark.rapids.shims.spark320
 import com.nvidia.spark.rapids.{SparkShims, SparkShimVersion}
 
 object SparkShimServiceProvider {
-  val VERSION320 = SparkShimVersion(3, 2, 0)
+  // temporarily allow 3.2.1 while 3.2.0 release candidates are being produced
+  val VERSION320 = SparkShimVersion(3, 2, 1)
   val VERSIONNAMES: Seq[String] = Seq(VERSION320)
```

Comment:
referencing version320 which was removed. Also I think we should leave 320 in there in case someone is using older snapshot.

Reply:
never mind on the name, realize VERSION320 just points to 3.2.1.  I do think we should leave 3.2.0 as well though

Reply:
@tgravescs I have updated this now